test/blackbox-tests/test-cases/deps-conf-vars.t/run.t

```diff
@@ -12,11 +12,9 @@ for this feature.
 
   $ dune build --root dynamic
   Entering directory 'dynamic'
-  File "dune", line 1, characters 0-43:
-  1 | (alias
-  2 |  (name default)
-  3 |  (deps %{read:foo}))
   Error: No rule found for foo
+  -> required by foo
```

Comment:
I guess these two `foo`s above refer to different things in the test, right? Perhaps, one of them should be renamed to make the test clearer.

Reply:
Yeah, let me see what's going on.

Reply:
I removed the duplication.